bin/run-ci.sh

```diff
@@ -5,10 +5,8 @@ set -o nounset
 sbt -Dfile.encoding=UTF-8 \
   -J-XX:ReservedCodeCacheSize=512M \
   -J-Xms1024M -J-Xmx2048M -J-server \
-  mimaReportBinaryIssues \
   scalafmtCheckAll \
   scalafmtSbtCheck \
-  whitesourceCheckPolicies \
```

Comment:
FYI we aren't using Whitesource at all anymore at Lightbend, so it would be fine to simply rip everything related to it out.

Reply:
Yea. I saw that in some other repo so I removed that line.

Reply:
I submitted the removal separately as #1032.